manifests/plugins/vmstat.pp

```diff
@@ -36,9 +37,13 @@
     require => Package['sysstat']
   }
 
-  collectd::plugins::plugin_common { 'vmstat':
-    package_name         => 'collectd-python',
-    plugin_file_name     => '10-vmstat.conf',
-    plugin_template_name => 'vmstat/10-vmstat.conf.erb',
+  collectd::plugin { 'vmstat':
+    package_name        => 'collectd-vmstat',
```

Comment:
This package doesn't exists and should be removed or changed to collectd-python. RHEL platforms break on this